### PR TITLE
Package splitting

### PR DESCRIPTION
**Pull request by gcotelli**
- Split packages and add groups to the baseline so Server and Client facilities can be loaded independently.
- Improved travis setup.
- Added contribution guidelines.
- Improved installation instructions to include usage as dependency and available groups purpose.

@juliendelplanque you need to enable the repo in coveralls.io interface before merging.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Julien Delplanque
+Copyright (c) 2018-2019 Julien Delplanque
```
Comment (juliendelplanque) [inline LICENSE:3]: Since you contributed, maybe you can add your name here?

**Comment by gcotelli** [comment]
BTW, I've issued hpi-swa/smalltalkCI#427 , because the coverage testing is not working on packages containing traits. So by now I've excluded this package for the coverage.

**Review by juliendelplanque — APPROVED** [review]
I approve but maybe you want to add your name in the LICENCE file?

File: LICENSE (hunk 1 above)
Comment (gcotelli) [reply LICENSE:3]: `2018-2019 Julien Delplanque and contributors` ? , just to not have to touch the LICENSE file every time someone makes a contribution.

Comment (gcotelli) [reply LICENSE:3]: IMHO just merge it as it is now. :)

Comment (juliendelplanque) [reply LICENSE:3]: Ok :-)

**Pull request merged**